README.md

```diff
@@ -6,11 +6,11 @@ This repo contains a simple neuro task to use as a starter for building [jspsych
 
 ## Getting Started
 
-1. Clone this repo onto your computer
+1. Clone this repo onto your computer using git
```

Comment:
should we call out that `project_name` is a placeholder?

Reply:
oh yes missed that! I will change that to `task-<TASK NAME>` instead and renaming in next step.